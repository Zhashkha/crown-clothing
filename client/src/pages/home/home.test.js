import React from "react";
import { shallow } from "enzyme";

import HomePage from "./home.component";

it("test HomePage snapshot", () => {
  expect(shallow(<HomePage />)).toMatchSnapshot();
});
