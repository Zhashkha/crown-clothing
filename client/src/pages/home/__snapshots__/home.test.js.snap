// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test HomePage snapshot 1`] = `ShallowWrapper {}`;
